plugins/osdn/common.go

```diff
@@ -159,26 +179,18 @@ func (oc *OvsController) validateNetworkConfig(clusterNetworkCIDR, serviceNetwor
 }
 
 func (oc *OvsController) StartMaster(clusterNetworkCIDR string, clusterBitsPerSubnet uint, serviceNetworkCIDR string) error {
-	// Any mismatch in cluster/service network is handled by WriteNetworkConfig
-	// For any new cluster/service network, ensure existing node subnets belong
-	// to the given cluster network and service IPs belong to the given service network
-	if _, err := oc.Registry.GetClusterNetworkCIDR(); err != nil {
-		subrange := make([]string, 0)
-		subnets, _, err := oc.Registry.GetSubnets()
-		if err != nil {
-			log.Errorf("Error in initializing/fetching subnets: %v", err)
-			return err
-		}
-		for _, sub := range subnets {
-			subrange = append(subrange, sub.SubnetCIDR)
-		}
+	var err error
```

Comment:
Consider naming the return type instead of declaring a new variable (e.g. func foo() (err error) {}).